### ISSUE TITLE
Store key in key store

### ISSUE THREAD

**Issue by gedw99** [znly/strobfus#3]
This is a nice until.

What about incorporate the option to store the key in the OS key store ?
There are a few golang libs that do this on all desktop and mobiles.

It's a nice addition I feel.

**Comment by Xe** [comment]
I don't think that makes sense for this library, given the point of it is to have the resulting binary work regardless of the fact that a value is encrypted.
